datamade/ccfp-asset-dashboard
# CIP Planner improvements for #29

## Overview
- Adds a CSV export for the CIP planner
- Changes the view that sends the data to the CIP planner
- Adds Phase as another column to the planner
- Sorting for the total score and total budget on the table
- Adds a button for each project in the planner, so a user can see the project detail page
- Refactors some styling and improved the look -- made the tables take up more of the blank space
- Reorganizes some of the file/folder locations for CSS/JS libraries

Closes issue #29.

### Demo
![planner-improvements](https://user-images.githubusercontent.com/38969506/107403005-392b2900-6aca-11eb-82cc-1b90b6ddb161.gif)

## Testing Instructions
- visit the review app deployment
- add a project
- update the project with a budget, a phase, scores, districts, and zones
- go to the cip planner. make sure all that data shows up in the table and the csv
- test the csv export
- test the sorting in the table
